### PR TITLE
updated dependencies

### PR DESCRIPTION
**Pull request by clarakosi**
Updated dependencies for swagger-router.

### CODE REVIEW

**Comment by Pchelolo** [comment]
Something is wrong with it - there was only one test that has been run. Probably some mocha changes between version 3 and 5 caused that.

**Comment by Pchelolo** [comment]
Note that during installation there's a warning from npm
```
npm WARN eslint-config-node-services@2.2.5 requires a peer of eslint-config-wikimedia@^0.5.0 but none was installed.
npm WARN eslint-config-node-services@2.2.5 requires a peer of eslint-plugin-jsdoc@^3.2.0 but none was installed.
npm WARN eslint-config-node-services@2.2.5 requires a peer of eslint-plugin-json@^1.2.0 but none was installed.
```

That's because the newer version of the services eslint configuration have some new peer dependencies (https://nodejs.org/en/blog/npm/peer-dependencies/) that should be explicitly specified in the package.json https://github.com/wikimedia/restbase/blob/master/package.json#L53-L56

That might be why only one test it completing.

**Review by d00rman — COMMENTED** [review]
Given all the changes in the deps, it would be also good to bump the patch version of the package.

File: package.json [hunk 1]
```diff
@@ -30,20 +30,23 @@
...
     "tassembly": "^0.2.3",
-    "template-expression-compiler": "^0.1.8",
-    "cassandra-uuid": "^0.0.2"
+    "template-expression-compiler": "^0.1.10",
+    "cassandra-uuid": "^0.1.0"
   },
   "devDependencies": {
-    "mocha": "^3.1.0",
+    "mocha": "^5.2.0",
     "mocha-jshint": "^2.3.1",
```
Comment (Pchelolo) [inline package.json]: Don't need to depend on it anymore.

File: package.json [hunk 2]
```diff
@@ -30,20 +30,23 @@
...
     "istanbul": "^0.4.5",
-    "mocha-lcov-reporter": "^1.2.0",
-    "coveralls": "^2.11.14",
+    "mocha-lcov-reporter": "^1.3.0",
+    "coveralls": "^3.0.2",
     "mocha-jscs": "^5.0.1",
```
Comment (Pchelolo) [inline package.json]: Idem

**Review by Pchelolo — COMMENTED** [review]
Can remove a couple of deps.

File: package.json (hunk 1 above)
Comment (clarakosi) [reply package.json]: They are peer dependencies of `eslint-config-node-services` and without both eslint fails

Comment (Pchelolo) [reply package.json]: I'm pretty sure that is not true https://github.com/wikimedia/eslint-config-node-services/blob/master/package.json#L26-L29

Comment (clarakosi) [reply package.json]: sorry I was thinking of `eslint-plugin-jsdoc`

**Comment by Pchelolo** [comment]
+1

**Comment by d00rman** [comment]
Tagged as [v0.7.2](https://github.com/wikimedia/swagger-router/tree/v0.7.2)

**Pull request merged**